app/uk/gov/hmrc/customs/declarations/information/services/declarationsInformationServices.scala

```diff
@@ -84,7 +80,7 @@ class DeclarationVersionService @Inject()(versionResponseFilterService: VersionR
       case "cds60002" => backendCDS60002MrnInvalidResponse
       case "cds60003" => backendCDS60003InternalServerErrorResponse
       case "cds60011" => backendCDS60011SubmissionChannelInvalidResponse
-      case _          => ErrorInternalServerError
+      case _ => ErrorInternalServerError
```

Comment:
can these have the same indention as before?

Reply:
Not sure how this happened, but will do.